collection_requirements.rst

```diff
@@ -189,6 +189,14 @@ Also:
 
 See `Migrating content to a different collection <https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#migrating-ansible-content-to-a-different-collection>`_ for complete details.
 
+After moving a module/plugin between collections:
+
+1) we deprecate the module/plugin with removal scheduled for the next major version once the following is satisfied:
+   - if the target collection is also contained in Ansible: if the target collection has been released with the new content as a new minor version for the same major version that is currently included in Ansible
+   - this ensures that users can resolve the deprecation warning without having to install something manually that potentially breaks backwards compatibility
```

Comment:
```suggestion
     - this ensures that users can resolve the deprecation warning without having to install something manually that potentially breaks backwards compatibility
```
I think this should be indented in, since it is not a condition, but explains the previous condition.

Reply:
ok, committed, thanks